tests/integration/netapi/rest_tornado/test_app.py

```diff
@@ -705,9 +730,7 @@ def verify_event(future):
                 )
             self.assertEqual(event["tag"], "salt/netapi/hook")
             self.assertIn("headers", event["data"])
-            self.assertEqual(
-                event["data"]["post"], {"foo": salt.utils.stringutils.to_bytes("bar")}
-            )
+            self.assertEqual(event["data"]["post"], {"foo": "bar"})
```

Comment:
The change here makes this test able to pass, but it still seems flaky in Jenkins, so I've left it skipping.  When it fails, it's with:

```text
Traceback (most recent call last):
  File "/tmp/kitchen/testing/tests/integration/netapi/rest_tornado/test_app.py", line 723, in test_post
    event = future.result()
  File "/tmp/kitchen/testing/salt/ext/tornado/concurrent.py", line 242, in result
    self._check_done()
Exception: DummyFuture does not support blocking for results

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/kitchen/testing/salt/ext/tornado/testing.py", line 137, in __call__
    result = self.orig_method(*args, **kwargs)
  File "/tmp/kitchen/testing/tests/integration/netapi/rest_tornado/test_app.py", line 727, in test_post
    resolve_future_timeout, exc
AssertionError: Failed to resolve future under 60 secs: DummyFuture does not support blocking for result
```